### DISCUSSION TITLE
To get AWS/GCP monthly bill based on group of users through Spod.

### DISCUSSION THREAD

**Discussion by braj065** [serverpod/serverpod#813] [Q&A] [answered]
Hi,

Is there a solution in AWS or GCP through which we can get the monthly server bill based on group of users. For example- If i have 8 users. 5 belong to company X and 3 belong to company Y. All the users access the server through client app on regular basis. Now at the end of month can i figure out what is the server bill for company X and for Y? All the user info and company details are stored in postgres. Hope my question makes sense.

**Comment by vlidholt** [comment, marked as answer]
It sounds like you will need to store some usage information with each company. Perhaps look at the number of requests they make to the server, or how many rows of data they have stored in the database.

**Reply by braj065** [reply]
Thanks @vlidholt
